src/MLJDecisionTreeInterface.jl

```diff
@@ -212,7 +284,18 @@ function MMI.fit(m::DecisionTreeRegressor, verbosity::Int, X, y)
         tree = DT.prune_tree(tree, m.merge_purity_threshold)
     end
     cache  = nothing
-    report = NamedTuple()
+
+    if m.feature_importance != :none
+        fi = feature_importance_options[m.feature_importance](tree)
+        fi_pairs = collect(Dict(zip(features, fi)))
+        # sort descending
+        sort!(fi_pairs, by= x->-x[2])
+        report = (feature_importances=fi_pairs,)
+    else
+        report = NamedTuple()
+    end
```

Comment:
put computation for feature importances in ` MMI.feature_importances` method. See comments above.